campfire-games.html

```diff
@@ -0,0 +1,240 @@
+        aria-controls="linkContainer" aria-expanded="false" aria-label="Toggle navigation">
+        <img class="navbar-toggler-icon" src="assets/img/hamburger.svg"></span>
+      </button>
+      <div id="linkContainer" class="navbar-collapse collapse">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" href="index.html#tracks">Tracks</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="index.html#schedule">Schedule</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="index.html#faq">FAQ</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="index.html#sponsors">Sponsors</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="campfire-games.html">Campfire Games</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <a class="mlh-trust-badge" id="mlh-trust-badge"
+      href="https://mlh.io/seasons/2021/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2021-season&utm_content=white"
+      target="_blank"><img src="https://s3.amazonaws.com/logged-assets/trust-badge/2021/mlh-trust-badge-2021-white.svg"
+        alt="Major League Hacking 2021 Hackathon Season"></a>
+  </nav>
+  <!-- End Navigation -->
+
+  <main class="main">
+    <!-- Begin "The Campfire Games" -->
+    <section id="intro" class="container">
+      <!-- set margin here -->
+      <h2 class="section-title">The Campfire Games</h2>
+      <p>
+        For this year's theme, Explore Together, The Campfire Games allow you to learn, grow and build with the Bitcamp Community.
+        You will belong to one of three teams, based on personality and interests, allowing you to meet and work with hackers from around the nation that share your passions.
+        By winning unqiue challenges, attending workshops and participating in mini-events, you'll be racking up points for your team, with the winning group receiving limited edition Bitcamp Apparel.
+        FInd your community, develop your team identity and collaborate on something bigger than yourself. #ExploreTogether
+      </p>
+    </section>
+    <!-- End "The Campfire Games" -->
+
+    <!-- Begin "Teams" -->
+    <section id="teams-container" class="container-team">
+      <section id="teams" class="container">
+        <h2 class="section-title">Teams</h2>
+        <div class="row">
+          <div class="col-md-3">
+            <center>
+              <img class="marshie" src="assets/img/marshie_blue.PNG" alt="Blue marshie">
+            </center>
+          </div>
+          <div class="col-md-9">
+            <p class="right-paragraph">
+              <br></br>
+              The <b>blue team</b> is rational and clear-headed. Their curiosity and intelligence are some of their greatest assets in understanding and exploring the world around them!
```

Comment:
Great work with the responsiveness of this section! A few points to consider:

1. Have you tried seeing how it looks when you make the `color` of each team name correspond to the color of that team? Think it might look good. So 
   - "blue team" would be #2875cc
   - "green team" would be #009050
   - "red team" would be #ff3f46
  
2. Most people prefer `<strong>` tags over `<b>` tags because of "semantic html" tags. Even though they both make text bold, the name of the `<strong>` tag more closely matches the type of content they contain.

3. Instead of using `<br>` tags for vertical alignment, you can instead use flexbox. If you want a good reference, I'd recommend [this CSS tricks article](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).  Flexbox is a little tricky to pick up, but here are the styles you'd apply. Essentially, we're telling the flexbox container to vertically center the content inside of it:
```css
.row .col-md-9 {
   display: flex;
   align-items: center;
}
```